username_api.py

```diff
@@ -31,6 +71,8 @@ def get_avatar(website, username):
     data = patterns['avatar'][website]
 
     if not data:
+        if DEBUG:
```

Comment:
we do not want or need to check for ``DEBUG`` variable when logging. 

Logging levels can be set in the beginning of the program. 
Also, check this out in order to avoid ``log_response()`` call and directly use ``logger`` to log into a file.

https://docs.python.org/3.6/howto/logging.html#logging-to-a-file

Reply:
yeah, I think we can move the `DEBUG` variable up.

I used `log_response` since we need both giving dump of the response and some msg for logging

Reply:
> Is there a way to log both to stdout and file?

Just need to add another handler: https://stackoverflow.com/a/40858741/1180321

Reply:
don't feel the need for wrapper. ``logging`` provides all the common use cases which we can set in the beginning of the file.

Reply:
I've just found using `Filter` class.
is this what you're thinking of?